### DISCUSSION TITLE
GitLens on an isolated network, avoiding pre-release versions.

### DISCUSSION THREAD

**Discussion by sullytWork** [gitkraken/vscode-gitlens#3194] [Q&A] [answered]
Hi, the team's working on an isolated network - one without any internet connection.   Our rocess is to downloaded GitLens as a VSIX extension onto the isolated network and then install it.  The process of getting GitLens onto the isolated network is length due to the nature of the network so we need to avoid doing it too often.

The issue we encounter with GitLens is the that we keep getting pre-leased versions that expire rapidly.  

_GitLens pre-release version ... has expired._

Wondering how we could get a version of GitLens as a VSIX extension that is not a pre-release and/or does not expire rapidly.  Ideally we are looking at a minimum of 6 months before needed to update.  

We've tried downloading multiple versions from the Marketplace, but all expired as "pre-releases" and all expired within weeks. 

Cheers

**Comment by eamodio** [comment, marked as answer]
You can download the non-"pre-release" versions from here: https://github.com/gitkraken/vscode-gitlens/releases which is what gets published to the marketplace

Unfortunately until the VS Code Marketplace starts supporting pre-releases (like they are inside VS Code) downloading the latest non-"pre-release" isn't a good experience as you'd have to "hack" the download URL yourself.
